graphbolt/src/cuda/neighbor_sampler.cu

```diff
@@ -471,7 +487,11 @@ c10::intrusive_ptr<sampling::FusedSampledSubgraph> SampleNeighbors(
                     "Selected edge_id_t must be capable of storing edge_ids.");
                 // Using bfloat16 for random numbers works just as reliably as
                 // float32 and provides around 30% speedup.
+#ifdef GRAPHBOLT_USE_HIP
+                using rnd_t = hip_bfloat16;
```

Comment:
We can't `#define nv_bfloat16 hip_bfloat16` somewhere?

Reply:
We can and should, I've fixed it and will push the changes shortly